features/dd-trace-core/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+import com.datadog.gradle.config.androidLibraryConfig
+import com.datadog.gradle.config.dependencyUpdateConfig
+import com.datadog.gradle.config.javadocConfig
+import com.datadog.gradle.config.junitConfig
+import com.datadog.gradle.config.kotlinConfig
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    // Build
+    id("com.android.library")
+    kotlin("android")
+    id("com.google.devtools.ksp")
```

Comment:
There is no Kotlin in this module, why this plugin is needed? Same for other Kotlin-related plugins and configurations in this file.

Reply:
Just a bad copy paste from others. Tks for noticing it.

Reply:
I see Kotlin is used in tests, but still, there is nothing here which needs KSP support, this line can be removed.